cornell-dti/course-plan
# Switch to new AP/IB infra

### Summary 






This pull request is step 4 in the AP/IB Infra Update #576.

- [x] switched from old infra to new infra 😄 
- [x] deprecated the `derivedAPIBEquivalentCourseData` implemented in #506 (it is no longer being used)


#### TODO
Now is probably a good time to write some e2e cypress tests -- new years resolution?? 🍾


### Test Plan 


AP/IB credit should not be broken 🙈 
![image](https://user-images.githubusercontent.com/34158284/147805102-484825df-9447-4527-828d-3cde8eb6894f.png)

Some flows to test:
- add exam that should satisfy some requirements
- add exam that shouldn't satisfy any requirements
- add multiple exams simultaneously
- remove exams
- check edges cases (like total credits requirement, FWS)
